### ISSUE TITLE
Select color from annotation color popup with number keys

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#3351]
https://forums.zotero.org/discussion/106834/position-the-color-selector-near-the-mouse-pointer-when-highlighting
https://forums.zotero.org/discussion/comment/441389/#Comment_441389

When the color popup is open, you should be able to press the number keys (no modifiers) to select the colors.

**Comment by max-alan-miller** [comment]
(On Zotero forum too, I can remove)

Thanks Dstillman! Is there any way to use this same shortcut to pick a color when selecting text? (in addition to when picking color for the highlighted tool)

What I usually do when note-taking:
1. Highlight text in a colour (requiring choosing/clicking colour each time)
2. Copy+Paste this into a single notes page (would also be great to have a keyboard shortcut for "Add to Note" feature)- as it's quicker to do that than click "Add to Note" for now
3. (I then try and reformulate my understanding of that citation in my own shorter words)

Hence, using the highlighter tool doesn't work for me here as I need to select that text to Copy-Paste it into a note. Would be good to have a shortcut to pick the colour at this moment (what you call the popup? under my understanding), doesn't seem like there is one right now (only to choose colour for the highlighter tool, not popup). 

Alternatively, some kind of toggle like "When using highlighter tool to highlight text, automatically send to Note" -> might solve everything. 

Let me know if this feedback should go anywhere else, I'm struggling to find a centralised point for feedback. Thanks for things so far!
